flink-python/pyflink/table/tests/test_table_completeness.py

```diff
@@ -37,7 +37,8 @@ def java_class(cls):
     @classmethod
     def excluded_methods(cls):
         # row-based operators should be supported when UDFs supported in python.
-        return {'map', 'flatMap', 'flatAggregate',  'aggregate'}
+        return {'map', 'flatMap', 'flatAggregate',  'aggregate', 'leftOuterJoinLateral',
+                'createTemporalTableFunction', 'joinLateral', 'getTableOperation', 'getSchema'}
```

Comment:
Can you add some comment for why we excluded `getSchema`?

Reply:
Done.